### DISCUSSION TITLE
A lot of functionality is not working on Samsung Galaxy S24 Ultra

### DISCUSSION THREAD

**Discussion by ZusPlay** [d4rken-org/capod#270] [Q&A]
A lot of functionality is not working on the Samsung Galaxy S24 Ultra (last update) for AirPods Pro 2 (USB-C), namely:

- Battery level cases.
- Ear detection with automatic play/pause.
- Show popup when case is opened.

Can you fix it?

**Comment by OMGMusicIsLifeee** [comment]
Samsung S22 with Pro 2s having the same issues. When I put my earbuds in the case it is able to read the case battery for a second. I belive their is an error with receiving case data.

**Comment by rohilv** [comment]
++ Samsung galaxy s23 ultra
